src/datasources/alarms/query_help.md

```diff
@@ -0,0 +1,27 @@
+### SystemLink Alarms data source
+
+The _SystemLink Alarms_ data source allows you to display alarm properties and alarm trend in dashboards and derive insights.
+
+### Key Features
+
+- **Display Alarm Properties**: Display various properties of your alarms directly in your dashboards. These properties include **Display Name**, **Channel**, **Active**, **Current Severity Level**, **Acknowledged**, and **Occurred At**.
+- **Display Alarm Count**: Display the total count of alarms based on the specific query conditions.
+- **Alarm Trend Analysis**: Visualize and monitor alarm trends over time, grouped by severity.
+- **Custom Queries**: Create custom queries with various filters to display alarm and their transition properties, retrieve alarm counts, or visualize alarm trends over time, matching your filter.
+
+### Example Queries
+
+You can use the _SystemLink Alarms_ data source for scenarios such as:
+
+- **List all active critical alarms**: Retrieve a list of all active alarms with critical severity level along with their properties.
+- **Count of unacknowledged alarms**: Retrieve and display a count of alarms that have not been acknowledged, filtered by various query conditions.
+- **View the trend of active alarms over the past week**: Visualize alarm activity over the past week to identify spikes and anomalies.
```

Comment:
Suggestion:
- '**View the trend of active alarms over the past week**: Visualize weekly alarm activity to identify anomalies.'